react-native-libraries.json

```diff
@@ -2345,7 +2345,11 @@
     "githubUrl": "https://github.com/JimmyDaddy/react-native-image-marker",
     "ios": true,
     "android": true,
-    "examples": ["https://github.com/JimmyDaddy/react-native-image-marker/tree/master/example"],
+    "expoGo": true,
```

Comment:
Hello @JimmyDaddy, thanks you for the update! 👍 

Also thank you for adding better Expo support, unfortunately your library will not work with Expo Go due to native files used. Config plugins are a part of `prebuild`, and their goal is to automate CNG processes for users. Expo Go ships with native files already bundled, and they cannot be extended.

We talk about it a bit more in the [Readme file](https://github.com/react-native-community/directory?tab=readme-ov-file#expogo), where we also explain why there is no "Expo" flag.

```suggestion
```

Reply:
I set it up according to react-native-vision-camera in libraries.json .😂